endpoints/openrtb2/amp_auction.go

```diff
@@ -387,6 +387,7 @@ func sendAmpResponse(
 	// Fixes #231
 	enc := json.NewEncoder(w)
 	enc.SetEscapeHTML(false)
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
```

Comment:
Why is it necessary to add this header set?

Reply:
it is done any way - added it explicitly to make clear that the text encoding is intended despite the JSON encoding (I assume due to the AMP requirements).  Would you prefer to remove it? @SyntaxNode

Reply:
@SyntaxNode and I discussed offline. You're correct in that `text/plain` is what PBS-Go returns. We're not sure why that is as it predates us and I can't find any documentation on the subject. I did notice that `application/json` is returned by PBS-Java in the 200 success case, which is probably what the content type should be. However, I suggest we keep the behavior as `text/plain` since it has been this way for years without complaint and keep this line of code but add a comment above it that is something like:
"Explicitly set content type to text/plain, which had previously been the implied behavior from the time the project was launched. It's unclear why text/plain was chosen or if it was an oversight, nevertheless we will keep it as such for compatibility reasons"